src/ngraph/pass/opset1_upgrade.cpp

```diff
@@ -75,6 +76,107 @@ bool pass::Opset1Upgrade::run_on_node(shared_ptr<Node> node)
 #endif
     switch (get_typeid(node))
     {
+    case OP_TYPEID::Convolution:
+    {
+        auto tmp = dynamic_cast<const op::v0::Convolution*>(node.get());
+        auto strides = tmp->get_window_movement_strides();
+        auto dilations = tmp->get_window_dilation_strides();
+        auto pads_begin = tmp->get_padding_below();
+        auto pads_end = tmp->get_padding_above();
+        auto data_dilation_strides = tmp->get_data_dilation_strides();
+        auto auto_pad = tmp->get_pad_type();
+
+        bool is_dds_valid = true;
+        for (auto value : data_dilation_strides)
+        {
+            is_dds_valid = is_dds_valid && (value == 1);
+        }
+
+        NGRAPH_CHECK(is_dds_valid,
```

Comment:
(Just thinking out loud...) this _would_ be possible if we had an "interior padding" operation, but we got rid of that. With DynReplaceSlice we could rig something up, but since data dilation was (AFAIK) totally unused anyway, it may not be worth the effort.

Reply:
We should avoid adding code that will not be used IMHO.